6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,24 @@
+// Function to calculate average of inputs with process
+function averageInput(){
+    const inputs = process.argv.slice(2)
+    let total = 0;
+    if (inputs.length < 3) {
+        console.log('Please type three numbers.');
+    } else {
+        let allNumbers = true;
+        inputs.forEach(input => {
```

Comment:
Do you think you could use another Array method? For example, one that runs a callback function on each element in the array and returns a boolean?